layout/includes/slide/script.pug

```diff
@@ -12,7 +11,11 @@ script(src=url_for("https://cdn.jsdelivr.net/npm/reveal.js@3.6.0/js/reveal.min.j
 - var parallaxBackgroundVertical = slide.parallaxBackgroundVertical || defaultSlide.parallaxBackgroundVertical
 - var autoSlide = slide.autoSlide || defaultSlide.autoSlide
 - var loop = slide.loop || defaultSlide.loop
-- var autoSlideStoppable = slide.autoSlideStoppable || defaultSlide.autoSlideStoppable
+- var controlsLayout = slide.controlsLayout || defaultSlide.controlsLayout
+- var controlsBackArrows = slide.controlsBackArrows || defaultSlide.controlsBackArrows
+- var progress = slide.progress || defaultSlide.progress
+- var showNotes = slide.showNotes || defaultSlide.showNotes
```

Comment:
defaultSlide is in `_config.yml`: https://github.com/Molunerfinn/hexo-theme-melody/blob/dev/_config.yml#L162-L174

But some people update the theme without update the `melody.yml` or `_config.yml` would get wrong.

So you need to add some fallback default value. Such as:

```
var controlsLayout = slide.controlsLayout || defaultSlide.controlsLayout || 'xxxx'
var showNotes = slide.showNotes || defaultSlide.showNotes || false

// etc
```

Reply:
And you need to add default value to the hexo-theme-melody's _config.yml just like above

Reply:
Thanks, I have added them all below